assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -6,48 +6,169 @@
 
 {% block details%}
 
-    <div class="row col-12 m-0 p-0 p-md-2 py-2 text-size-3">
-        <div class="{% if can_delete_all or can_delete_some %} col-10 col-sm-11 {% else %} col-12 {% endif %}">
-            <div class="row pr-2">
-                <div class="col-12 col-md-6 p-0 show-md"><b>Name</b></div>
-                <div class=" text-center col-12 col-md-6 d-none d-md-flex">
-                    <div class="col-3 p-0"><b>Shopped</b></div>
-                    <div class="col-3 p-0"><b>Cooked</b></div>
-                    <div class="col-3 p-0"><b>Cleaned</b></div>
-                    <div class="col-3 p-0"><b>Paid</b></div>
+    <div class="row">
+        <form method="post" class="col-12">
+            {% csrf_token %}
+            <div class="row">
+                <div class="col-12 table-responsive">
+                    <table class="table table-hover">
+                        {# Table header #}
+                        <thead>
+                        <tr class="">
+                            <td style="min-width: 10rem">Name</td>
+                            <td style="min-width: 15rem;">Help stats</td>
+                            {% if show_delete_column %}
+                                <td>Delete</td>
+                            {% endif %}
+                        </tr>
+                        </thead>
+
+                        <tbody>
+                        {# Table rows #}
+                        {% for entry in entries %}
+                            <tr>
+                                <td>
+                                    {{ entry.name }}
+                                    {% if entry.get_external %}
+                                        <span class="small font-italic">added by {{ entry.user }}</span>
+                                    {% endif %}
+                                </td>
+                                <td>
+                                    <div class="text-center btn-group btn-group-toggle d-flex" data-toggle="buttons" style="width:100%;">
+                                        {% with entry.get_internal as entry_data %}
+                                            {% if entry_data %}
+                                                {% if can_edit_stats %}
+                                                    <label class="btn btn-outline-primary col-3
```

Comment:
Like I mentioned, in this case we're inside a table cell and these don't have fixed widths which requires a little tweaking to get the bootstrap input group working correctly by themselves without using the custom flex layout. It's really helpful to use the classes only in the way that they're meant to be used, to prevent unwanted side effects for now and also in the future. But as always it's just a note, if you want to know how it could be done I could show you, otherwise leave it like it is now.